pkg/scheduler/framework/runtime/framework.go

```diff
@@ -346,12 +330,85 @@ func NewFramework(r Registry, profile *config.KubeSchedulerProfile, opts ...Opti
 		fillEventToPluginMap(p, options.clusterEventMap)
 	}
 
+	// initialize plugins per individual extension points
 	for _, e := range f.getExtensionPoints(profile.Plugins) {
 		if err := updatePluginList(e.slicePtr, *e.plugins, pluginsMap); err != nil {
 			return nil, err
 		}
 	}
 
+	// initialize MultiPoint plugins
```

Comment:
just to be extra safe and ensure we are not changing the behavior for v1beta2:

Put this logic in a separate function and within that do at the beginning of the function:

```golang

if len(Plugins.Multipoint.Enabled) == 0 {
  return
}
```

Reply:
Made the change, but instead of putting the check in the function I wrapped the function call in the check to fail faster. Is there a preference to returning within the function call instead?